homeassistant/components/mazda/__init__.py

```diff
@@ -67,12 +70,14 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         _LOGGER.error("Error occurred during Mazda login request: %s", ex)
         raise ConfigEntryNotReady from ex
 
-    async def async_handle_service_call(service_call=None):
+    async def async_handle_service_call(service_call: ServiceCall) -> None:
         """Handle a service call."""
         # Get device entry from device registry
         dev_reg = device_registry.async_get(hass)
         device_id = service_call.data["device_id"]
         device_entry = dev_reg.async_get(device_id)
+        if TYPE_CHECKING:
+            assert device_entry
```

Comment:
It has already been checked in the schema validation (`validate_mazda_device_id`)
https://github.com/home-assistant/core/blob/41c497ee6e4d32e3316af653f5f03b3ba679eb9c/homeassistant/components/mazda/__init__.py#L115-L116